.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+README.md
```

Comment:
Why is this ignored?

Reply:
Will be taken care of when I am ready to write the README.md